client/state/login/actions.js

```diff
@@ -13,6 +13,8 @@ import {
 	LOGIN_REQUEST,
 	LOGIN_REQUEST_FAILURE,
 	LOGIN_REQUEST_SUCCESS,
+	LOGIN_TWOFACTOR_PUSH_POLL_START,
+	LOGIN_TWOFACTOR_PUSH_POLL_STOP,
```

Comment:
This happens a few lines below, but we should remove any occurrence of `empty_two_step_code` since the server no longer return such error code.